### ISSUE TITLE
Why the value of true.npy is not the same as the original value?

### ISSUE THREAD

**Issue by gaofenxia** [salesforce/ETSformer#7]
Hello, thank you very much for your work. I have a few unclear points about the code：
（1）Why is the value of true.npy saved in the exp_main.py file not the same as the corresponding original value?
（2）How and where do I print the training set and test set?

**Comment by gorold** [comment]
Hi, thanks for the interest. Could you elaborate more about what your difficulties are? (1) What is not the same? (2) Where do you want to print it? The datasets are likely too large to print meaningfully. Perhaps you want to visualize them as plots?
